src/versionbits.h

```diff
@@ -37,7 +37,7 @@ typedef std::map<const CBlockIndex *, ThresholdState> ThresholdConditionCache;
 struct ForkDeploymentInfo
 {
     /** Deployment name */
-    char *name; // bip135: removed const to allow update from CSV
+    const char *name; // bip135: removed const to allow update from CSV
     /** Whether GBT clients can safely ignore this rule in simplified usage */
```

Comment:
isn't this supposed to be non const?  says in the comment it needs to be non const so we can update from csv...

Reply:
well, that's interesting.  The const here refers to the contents of the item pointed to, not the pointer itself.  The code in forks_csv.cpp would be very broken if it modified the contents because it is initialized as an empty string.  In fact, the code does just modify the pointer:
forks_csv.cpp: 139
                // copy the data
                if (name.length()) {
                    char* c = new char[name.length() + 1];
                    strncpy(c, name.c_str(), name.length());
                    c[name.length()] = 0;
                    vbinfo.name = c;
                }

I will remove the comment.